dvc/config_schema.py

```diff
@@ -145,6 +145,7 @@ class RelPath(str):
                     "session_token": str,
                     Optional("listobjects", default=False): Bool,  # obsoleted
                     Optional("use_ssl", default=True): Bool,
+                    Optional("verify", default=True): Bool,
```

Comment:
`verify` is already used for different purpose (see REMOTE_COMMON above), so we need to use something else. E.g.
```suggestion
                    Optional("verify_ssl", default=True): Bool,
```
?